Lesson39/savas/src/components/__test__/BackgroundColorChanger.test.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { render, fireEvent } from "@testing-library/react";
+import BackgroundColorChanger from "../BackgroundColorChanger";
+
+test("Background color changes on button click", () => {
+  const { getByTestId, container } = render(<BackgroundColorChanger />);
```

Comment:
I don't see a good reason to extract getByTestId and container from render method. I'd recommend following official guide, see example here: https://testing-library.com/docs/react-testing-library/example-intro#full-example

Reply:
Were these changes appropriate?